public/javascripts/floatingDots.js

```diff
@@ -135,7 +135,7 @@ function makeDots() {
     (function() {
       'use strict';
       canvas.width = $('body').innerWidth();
-      canvas.height = $('body').innerHeight() - 90;
+      canvas.height = $(window).height() - 77;
```

Comment:
generally a bad idea to have this number like this - called a `magic number`

Please use a named variable for brevity